template/package.json

```diff
@@ -17,13 +17,15 @@
     "postinstall": "bash ./scripts/post-install.sh",
     "reverse": "adb reverse tcp:8081 tcp:8081",
     "generateimages": "node scripts/genimg src/assets/images --output src/themes/image.js --name Images",
-    "generatefonts": "node scripts/fonts"
+    "generatefonts": "node scripts/fonts",
+    "plop": "plop"
   },
   "dependencies": {
     "@react-navigation/native": "^6.0.13",
     "@react-navigation/native-stack": "^6.9.1",
     "@reduxjs/toolkit": "^1.8.6",
     "i18n-js": "^4.1.1",
+    "plop": "^3.1.1",
```

Comment:
This library should add to `devDependencies` instead.

Reply:
Updated